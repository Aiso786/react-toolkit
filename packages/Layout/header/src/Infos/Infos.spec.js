import React from 'react';
import { shallow } from 'enzyme';
import Infos from './Infos';

describe('<Infos>', () => {
  it('renders Infos correctly', () => {
    const wrapper = shallow(
      <Infos infos={[{ word: 'word', definition: 'definition', id: '1' }]} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
