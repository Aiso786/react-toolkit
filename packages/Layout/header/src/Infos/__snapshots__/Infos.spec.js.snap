// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Infos> renders Infos correctly 1`] = `
<div
  className="af-contrat"
>
  <i
    className="glyphicon glyphicon-info-sign"
  />
  <dl
    className="af-contrat__list"
  >
    <dt
      className="af-contrat__word"
      key="word_1"
    >
      word
    </dt>
    <dd
      className="af-contrat__def"
      key="def_1"
    >
      definition
    </dd>
  </dl>
</div>
`;
